docs/manual/introduction/How-to-run-thing-locally.html

```diff
@@ -26,10 +26,10 @@
 
 			<ol>
 				<li>
-					Change security for local files in a browser (this allows you to access your page as <code>file:///yourFile.html</code>
+					Change security for local files in a browser, this allows you to access your page as <code>file:///yourFile.html</code>
```

Comment:
Change security for local files in a browser. This allows you to access your page as: